src/cljs/org/broadinstitute/firecloud_ui/paths.cljs

```diff
@@ -45,5 +45,8 @@
 (defn submission-details [workspace-id submission-id]
   (str (submissions-list workspace-id) "/" submission-id))
 
+(defn get-abort-submission-path [workspace-id submission-id]
+  (submission-details workspace-id submission-id))
+
```

Comment:
I first created this file and started us down this pattern.  My thinking was that it's a way of documenting which endpoint you're hitting specifically.  At the same time though, I didn't chain them together like this.  My thought was that it's essentially a coincidence that getting a thing and deleting the thing use the same URL.  Then again I'm not that experienced with REST so if this is an idiomatic enough thing I guess it makes sense to merge them.

I want to do a refactor to unify the paths and the mock data.  So when making an AJAX call, you'd just specify the endpoint and it would get the path and the mock data together.  That would go along with this idea of separate functions.